tools/depends/target/ffmpeg/Makefile

```diff
@@ -26,7 +26,10 @@ ifeq ($(CROSS_COMPILING), yes)
 endif
 ifeq ($(OS), linux)
   ffmpg_config += --target-os=$(OS) --cpu=$(CPU)
-  ffmpg_config += --enable-vdpau --enable-vaapi --enable-pic
+  ifneq ($(TARGET_PLATFORM),raspberry-pi)
+    ffmpg_config += --enable-vdpau
+  endif
+  ffmpg_config += --enable-vaapi --enable-pic
```

Comment:
I don't know if it silently passes --enable-vdpau or this also needs to be ecluded on rbpi, but what I know for sure is, it silently passed --enable-vdpau in version 3.1 and now it fails while searching for vdpau headers.